src/main/java/seedu/momentum/logic/parser/ParserUtil.java

```diff
@@ -54,8 +59,34 @@ public static Name parseName(String name) throws ParseException {
      */
     public static Description parseDescription(String description) {
         requireNonNull(description);
-        String trimmedAddress = description.trim();
-        return new Description(trimmedAddress);
+        String trimmedDescription = description.trim();
+        return new Description(trimmedDescription);
+    }
+
+    /**
+     * Parses {@code Optional<String> date} and {@code Optional<String> time}into a {@code Deadline}.
+     * Leading and trailing whitespaces will be trimmed.
+     */
+    public static Deadline parseDeadline(Optional<String> date, Optional<String> time) throws ParseException {
+        if (date.isEmpty() || date.get().isBlank()) {
```

Comment:
Should this throw an exception instead?

Reply:
This is to show deletion of a deadline so the deadline "created" is an empty one.